data/json/flags.json

```diff
@@ -83,6 +83,18 @@
     "info": "As a weapon, this item is <good>well-made</good> and will <info>withstand the punishment of combat</info>.",
     "conflicts": [ "FRAGILE_MELEE" ]
   },
+  {
+    "id": "EATEN_COLD",
+    "type": "json_flag",
+    "context": [ "COMESTIBLE" ],
+    "info": "This item tastes <good>better</good> while <info>cold</info>."
+  },
+  {
+    "id": "EATEN_HOT",
+    "type": "json_flag",
+    "context": [ "COMESTIBLE" ],
+    "info": "This item tastes <good>better</good> while <info>hot</info>."
```

Comment:
Might be more visually stimulating if `<info>hot</info>` was changed to `<color_red>hot</color>`

![subjectiveprwithhotcoldcolors](https://user-images.githubusercontent.com/32253888/43860043-b3e74a8a-9b0f-11e8-8e50-476baa530633.png)